clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -894,6 +895,51 @@ Syscall mix tool results:
               1 :       273
 \endcode
 
+\section sec_tool_record_filter Record Filter
+
+The record filter tool modifies a target trace.  It contains several varieties of
+filters which selectively remove records from the tool.  The filters currently provided
+include:
+
+- Removing records of types specified by the -filter_trace_types option.
+  The types are identified by their #dynamorio::drmemtrace::trace_type_t
+  enum numeric value.
+
+- Remove marker records of marker types specified by the -filter_marker_types option.
+  The types are identified by their #dynamorio::drmemtrace::trace_marker_type_t
+  enum numeric value.
+
+- Running a simple data cache filter and removing hits.  The cache is enbabled
+  and its size specified by the -filter_cache_size option.  The cache filter
+  can be applied only to the start of a trace using the -filter_stop_timestamp
```

Comment:
filter_stop_timestamp applies to all filters doesn't it?

Reply:
True, though I thought you would normally only use it with the cache filter.  I moved it to its own statement after this list.